programming/sr/motors/index.md

```diff
@@ -6,72 +6,70 @@ title: Motors
 Motors
 ======
 
-The `motors` object is used to control a collection of Motor Boards.
-Similar to `ruggeduinos` and `servos`, `motors` can be used like a list.
-To do something with the **first Motor Board**, you would use:
+The `motor_board` object is used to control a collection of Motor Boards.
+
+When a single Motor Board is connected to your robot, you can control it
+using the `motor_board` object.
 
 ~~~~~ python
-R.motors[0].something...
+R.motor_board.something...
 ~~~~~
 
-...because indexes are 0-based (counting starts from 0, not 1).
-
-When you have more than one Motor Board connected to your kit they will be ordered based upon their SR Part Code, as written on the underside of the board.
-
-The SR Part Code of each detected Motor Board is also printed to the log when your robot starts.
+The serial number of each detected Motor Board is printed to the log when your robot starts.
 It will look something like this:
 
 ~~~~~ not-code
-Found the following devices:
- - Motors:
-    0: Motor( serialnum = "SR0XJ1F" )
-    1: Motor( serialnum = "SR0A123" )
+sr.robot3.robot INFO - Found Student Robotics Motor Board v4 - srABC1
 ~~~~~
 
-
-However, like `ruggeduinos` and `servos`, `motors` is actually a dictionary.
-As a result, in `motors` you can also use the SR Part Code of the Motor Board as a key.
-For example, if you had a board that was labelled "SR0A123",
+If you have more than one Motor Board attached, the `motor_boards` object can be used to control a collection of Motor Board. It is a dictionary access via serial number, which is usually written on the board. For example, if you had a board whose serial number was "srABC1",
```

Comment:
Not sure "dictionary access" is going to mean a lot to competitors? The first sentence here also feels like it duplicates the initial paragraph quite a bit (not sure if that's intentional?)

Perhaps something like this would be simpler:
```suggestion
If you have more than one Motor Board attached, you need to specify which one you want to control. This is done using the part number of the board. For example: if you had a board that was labelled "srABC1",
```